include/highfive/H5PropertyList.hpp

```diff
@@ -64,9 +66,6 @@ class PropertyListBase: public Object {
         return plist;
     }
 
-  protected:
-    using Object::Object;
-
   private:
     template <typename T, typename U>
     friend T get_plist(const U&, hid_t (*f)(hid_t));
```

Comment:
What if you moved these lines to the `PropertyList`, would that enable `T{hid}`? (Maybe friendliness isn't inherited in old/some compilers?

Reply:
I don't want to put this function in the public API of property list.